Sources/Classes/core/MiniAppAnalytics.swift

```diff
@@ -38,6 +38,7 @@ internal enum MiniAppRATEvent: String, CaseIterable {
     case getSecureStorageSize = "mini_app_secure_storage_size"
     case setCloseAlertInfo = "mini_app_close_alert_info"
     case sendJsonToHostapp = "mini_app_send_json_to_host_app"
+    case closeMiniApp = "close_mini_app_with_confirmation_alert"
 
```

Comment:
Guess we are missing analytics for sendJsonToMiniapp method as well right ?

Reply:
Thank you I have updated in the latest commit.